client/src/pages/CalendarPage/CalendarPage.js

```diff
@@ -32,10 +32,10 @@ const dailyTaskDateFormatter = (date) => {
 };
 
 const CalendarPage = () => {
-  const [showModal, setModal] = useState(false);
+  const [showTaskDetails, setShowDetails] = useState(false);
```

Comment:
The set function should be renamed as `setShowTaskDetails` instead of `setShowDetails`

Reply:
Renamed.